lib/bolt/executor.rb

```diff
@@ -170,6 +170,7 @@ def run_task(targets, task, arguments, options = {}, &callback)
       log_action(description, targets) do
         notify = proc { |event| @notifier.notify(callback, event) if callback }
         options = { '_run_as' => run_as }.merge(options) if run_as
+        options['_task'] = task.name
```

Comment:
I think we want to add this to `arguments`, not to `options`. The options hash is for parameters that affect *how* the task is run, not for parameters that are passed *to* the task. Right now, this seems like it just passes `_task` to the transport, which ignores it.